inc/templates/namespace.php

```diff
@@ -524,6 +527,44 @@ function setup_head(
 	return $data;
 }
 
+/**
+ * Manage any active integrations by inserting an `irving/integrations` component.
+ *
+ * @param array $data Data object to be hydrated by template.
+ * @return array The update endpoint data.
+ */
+function setup_integrations( $data ) {
+	// Get any set integrations options.
+	$options = get_option( 'irving_integrations' );
+
+	// If options are present, append the integrations component into the endpoint response.
+	if ( ! empty( $options ) ) {
```

Comment:
Totally not a blocker, but this would read more clearly and avoid a nesting level by returning early here if `$options` is empty, i.e.

```php
// Bail early if no options are present.
if ( empty( $options ) ) {
	return $data;
}

// The rest of the logic here...
```